node/src/Pos/Block/Logic/Header.hs

```diff
@@ -164,8 +172,15 @@ classifyHeaders inRecovery headers = do
     tipHeader <- DB.getTipHeader @ssc
     let tip = headerHash tipHeader
     haveOldestParent <- isJust <$> DB.blkGetHeader @ssc oldestParentHash
-    let headersValid = isVerSuccess $
-                       verifyHeaders (headers & _Wrapped %~ toList)
+    leaders <-
+        lrcActionOnEpochReason
+            newestHeaderEpoch
+            (sformat ("classifyHeaders: there are no leaders for epoch " %build)
+                     newestHeaderEpoch)
```

Comment:
Putting information about epoch into reason is probably 100% useless because `LrcDataUnknown` has `EpochIndex` inside and is formatted this way: 
```
    build (LrcDataUnknown epoch reason) =
        bprint
            ("LRC data isn't presented for epoch #"%int%
             " so raise the exception with reason: "%stext)
            epoch reason
```

Reply:
Thanks, will fix.